docs/mms_on_fargate.md

```diff
@@ -0,0 +1,256 @@
+
+## Create LoadBalancer
+
+AWS Supports several different types of Load Blanacers:
+
+
+* Application Load Balancer, that works on the level 7 of the OSI model (effectively with the HTTP/HTTPS protocols)
+* TCP Load Balancer and
+* deprecated Elastic Load Balancer, which also used to work on the TCP level
+
+For our cluster we are going to use application load balancer.
+1. Open AWS console
+2. Goto EC2
+3. Goto to “Load balancers” section
+4. Create new Load Balancer
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/1__Create_Load_Balancer.png)
+
+5. Choose “HTTP/HTTPS”
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/2__HTTP_HTTPS+.png)
+
+6. Set all the required details. Most importantly set the VPC. VPC is very important. Having wrong VPC might cause our LB to not be able to communicate with your tasks. So memorize VPC that you going to use here for later.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3_2_Listeners_and_AZ+.png)
+
+7. Next is security group. This is also important. Your security group should:
+
+* allow inbound connection for port 80 (since this is the port on which LB will be listening on)
+* member of the security group should be able to talk to the security group where you planing to have your service. In our case we will use same security group everywhere in order to simplify our life.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/4.+Configure+Security+groups.png)
+
+8. Routing configuration is simple. The thing is here you need to create what is called “target group”. But the thing is, ECS will create required service group later on for you. Therefore now you need to create dummy “target group” that you will delete after the creation of the LB. Yes, I know, there should be option to skip this, but there is not :(
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/5.+Configure+Routing+(DUmmy).png)
+
+9. nothing need to be done on the last two steps. Finish the creation and ...
+10. now you are ready to remove dummy listener and target group
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/8__Delete_the_dummy_listener.png)
+
+Finally we are done-done-done with the LB, time to move to the service creation
+
+## Create a ECS Service from the ECS Task Definitions
+
+1. Go to Elastic Container Service → Task Definitions and select the task definitions name. Click on actions and select create service.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/1.+Go+to+task+definitions.png)
```

Comment:
don't forget image descriptions!

Reply:
I think the points are the image descriptions. Do you think we should add more description?